substrate-node/support/src/types.rs

```diff
@@ -69,7 +69,7 @@ pub struct FarmingPolicyLimit {
 }
 
 #[derive(PartialEq, Eq, PartialOrd, Ord, Clone, Encode, Decode, Default, Debug, TypeInfo)]
-pub struct Node<Location, PubConfig, If, SerialNumber> {
+pub struct Node<Resources, Location, PubConfig, If, SerialNumber> {
```

Comment:
Now `Resources` comes from a generic parameter of `Node`

Reply:
See above comment